tomviz/OperatorPython.h

```diff
@@ -32,6 +32,13 @@ class OperatorPython : public Operator
   Q_OBJECT
 
 public:
+  // Pass this constant in to EditOperatorWidget::setViewMode to change the tab
+  // to the Script tab where the python operator's code is viewable.  This only
+  // applies to the custom EditOperatorWidget used by OperatorPython and
+  // returned
+  // from its getEditorContents method.
+  static const QString VIEW_CODE_MODE;
```

Comment:
Why are we adding this to the public class interface? Is it accessed elsewhere?

Reply:
Yes, we are using it in the OperatorPropertiesPanel when the "View Code" button is pressed.

Reply:
type/constant:  I thought about making it an `enum`, but since the valid values vary from widget to widget, it didn't seem to fit.  Given that an `enum` didn't make sense I went with string over integer since they are more descriptive.  I guess the constant may be overkill, but if some string in two parts of the program have to match I try to make it a constant somewhere rather than relying on it staying the same.

My thoughts on the name: I didn't really like it either... but I could see reusing this to do more than set the default tab (for example change styling based on if we are in a dialog or the properties panel).

Reply:
Let's just keep it simple, and do string matching. I think this is a little over-engineered. Signals and slots are all based upon string matching at runtime, with some decoration. We can revisit this when/if it really needs more infrastructure, at that point I am more inclined to consider using JSON objects, and have been experimenting with that elsewhere.